modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/SlaCard/Inputs/index.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import ClayCard from '@clayui/card';
+import ClayLabel from '@clayui/label';
+import classNames from 'classnames';
+import React from 'react';
+
+const SlaCardLayout = ({
+	slaDateEnd,
+	slaDateStart,
+	slaLabel,
+	slaSelected,
+	slaTitle,
+}) => {
+	const slaDate = `${slaDateStart} - ${slaDateEnd}`;
+
+	return (
+		<div
+			className={classNames('align-items-center d-flex', {
+				'sla-card': slaLabel !== slaSelected,
+				'sla-card-active': slaLabel === slaSelected,
+			})}
+		>
+			<ClayCard
+				className={classNames('m-0 p-3 rounded-lg', {
+					'bg-brand-secondary-lighten-6 sla-gold':
+						slaTitle === 'Gold',
+					'bg-neutral-0 sla-limited': slaTitle === 'Limited',
+					'sla-platinum': slaTitle === 'Platinum',
```

Comment:
You are using these title strings many times in this component, I think that would be better and more easy to maintain 
 if it was set in an object, what do you think?

Reply:
Ok, @igordmf . Done!